atom/renderer/api/lib/desktop-capturer.coffee

```diff
@@ -0,0 +1,22 @@
+ipc = require 'ipc'
+remote = require 'remote'
+NativeImage = require 'native-image'
+
+EventEmitter = require('events').EventEmitter
+desktopCapturer = new EventEmitter
+
+# Tells main process the renderer is requiring 'desktop-capture' module.
+ipc.sendSync 'ATOM_BROWSER_DESKTOP_CAPTURER_REQUIRED'
```

Comment:
This message seems redundant, we can just do the initialization work in `startUpdating`.